src/Settings/Model/CustomsSettings.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\Pdk\Settings\Model;
+
+use MyParcelNL\Pdk\Base\Model\Model;
+use MyParcelNL\Pdk\Form\Model\Input\Select\CountrySelectInput;
+
+/**
+ * @property string $defaultFrom
+ * @property string $defaultCustomsCode
+ * @property string $defaultCountryOfOrigin
+ */
+class CustomsSettings extends Model
+{
+    public const DEFAULT_COUNTRY_OF_ORIGIN = 'defaultCountryOfOrigin';
+    public const DEFAULT_CUSTOMS_CODE      = 'defaultCustomsCode';
+    public const DEFAULT_FROM              = 'defaultFrom';
+
+    protected $attributes = [
+        self::DEFAULT_FROM              => null,
+        self::DEFAULT_CUSTOMS_CODE      => null,
+        self::DEFAULT_COUNTRY_OF_ORIGIN => CountrySelectInput::class,
+    ];
+
+    protected $casts      = [
+        self::DEFAULT_FROM              => 'string',
+        self::DEFAULT_CUSTOMS_CODE      => 'string',
+        self::DEFAULT_COUNTRY_OF_ORIGIN => CountrySelectInput::class,
+    ];
```

Comment:
Ik mis hier Type douanezending?
`<option value="1" selected="selected">Commerciële goederen</option>
<option value="2">Commerciële monsters</option>
<option value="3">Documenten</option>
<option value="4">Geschenken</option><option value="5">Retourzending</option>`

Reply:
Verwerkt in CustomsSettingsView, de options worden pas opgemaakt bij het opstellen van de view.